### PR TITLE
fix ruff invocation in linting workflow

### PR DESCRIPTION
**Pull request by egpbos**
Title says it all. The linting workflow is broken, this fixes it (I hope).

### CODE REVIEW

**Comment by egpbos** [comment]
Note that the torchtext failures seem unrelated as mentioned in #827. The only workflow that matters is the one that actually does work again now: the linting one :) https://github.com/dianna-ai/dianna/actions/runs/10145802048/job/28052379417?pr=828

File: pyproject.toml [hunk 1]
```diff
@@ -22,6 +22,35 @@ extras = dev
...
+    "_build",
+    "buck-out",
+    "build",
+    "dist",
+    "node_modules",
+    "venv",
+    ".venv",
+    "scripts",
+]
+
```
Comment (stefsmeets) [inline pyproject.toml]: Most of these are already in the defaults, I recommend to use `extend-exclude` instead (https://docs.astral.sh/ruff/settings/#extend-exclude):

```suggestion
extend-exclude = [
    ".venv",
    "scripts",
]

```

**Review by stefsmeets — APPROVED** [review]
Looks good to me! 🚀

**Comment by egpbos** [comment]
`.venv` is also in default exclude it seems, so also removed that in the force pushed amended commit.

**Pull request merged**